# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/infrastructure/safe-passing-of-structured-data/structuredclone_0.html.ini
# Commit: 0f9f2acf6e13
# Full Hash: 0f9f2acf6e136933da9a105df64ca4cc20bcb15c
# Author: Kagami Sascha Rosylight <krosylight@mozilla.com>
# Date: 2022-04-20 03:39:48
# Regressor Bug: 1561357
# File Overlap Count: 1
# Description:
#   Bug 1561357 - Implement [Serializable] for DOMException r=smaug,sfink
#   
#   Skipping stack serialization here as
#   
#   * it's optional per the spec
# ==============================================================================

diff -r 6e296387dc26 -r 0f9f2acf6e13 testing/web-platform/meta/html/infrastructure/safe-passing-of-structured-data/structuredclone_0.html.ini
--- a/testing/web-platform/meta/html/infrastructure/safe-passing-of-structured-data/structuredclone_0.html.ini	Tue Apr 19 19:13:08 2022 +0000
+++ b/testing/web-platform/meta/html/infrastructure/safe-passing-of-structured-data/structuredclone_0.html.ini	Tue Apr 19 19:31:27 2022 +0000
@@ -5,15 +5,9 @@
   [Error.message: getter is ignored when cloning]
     expected: FAIL
 
-  [DOMException objects created by the UA can be cloned]
-    expected: FAIL
-
   [EvalError objects can be cloned]
     expected: FAIL
 
-  [DOMException objects can be cloned]
-    expected: FAIL
-
   [URIError objects can be cloned]
     expected: FAIL
 
